addon-test-support/index.js

```diff
@@ -1,10 +1,13 @@
 import { find, triggerEvent } from '@ember/test-helpers';
 import { assert } from '@ember/debug';
 
-export async function upload(selector, file) {
+export async function upload(selector, ...files) {
   let input = find(selector);
   assert(`Selector '${selector}' is not input element.`, input.tagName === 'INPUT');
-  assert(`File must be instance of File/Blob type`, file instanceof Blob);
+  assert(
+    `All files must be instances of File/Blob type`,
+    files.every((file) => file instanceof Blob)
+  );
 
-  return await triggerEvent(input, 'change', { files: [file] });
+  return triggerEvent(input, 'change', { files });
```

Comment:
If I got it right, this adds support for testing upload of multiple files isn't it? Very helpful! :+1: Should document that. But as #275 rewrites the testing docs we might want to do it in another up following PR to avoid merge conflicts?

Reply:
Yeah, that's right. I figured multiple file selection is something we want to test since it's a documented feature.

It doesn't break the existing API though, which is nice.

If we can get #275 merged before this I'll update the docs in this PR.